### PR TITLE
Feature/simgan support/issue 232 fid

### PR DESCRIPTION
**Pull request by anishthite**
@roddtalebi one thing im not sure of is if I'm matching the right FID score to the right R, I think I am but idk maybe its shuffling it somehow

### CODE REVIEW

File: slurm_files/icehammer_simgan.sh [hunk 1]
```diff
@@ -7,8 +7,6 @@
 #SBATCH -t 0-08:00                                   # job will run at most 8 hours D-HH:MM
 #SBATCH --mem=128gb                                  # memory limit
 #SBATCH --output=simgan.out # output file relative to ezcgp folder
```
Comment (anishthite) [inline slurm_files/icehammer_simgan.sh:9]:
```suggestion
#SBATCH --output=simgan.out # output file relative to ezcgp folder
#SBATCH --mail-type=ALL                              # Will send a status email based on any combination of a,b,e
#SBATCH --mail-user=rtalebi3@gatech.edu              # Where to send email updates to
```

Comment (anishthite) [reply slurm_files/icehammer_simgan.sh:9]: fixing the email thing from before

**Comment by roddtalebi** [comment]
@anishthite I had that issue but fixed it and I thought maybe your branch was created before the fix but it looks the same in the 'files changed'...hmmm.
Is your [line 124](https://github.com/ezCGP/ezCGP/pull/237/files#diff-a97ccb8599becc1209701c40ab2a0ec1c167f11911abaea055690ca964dd560fR124) where you think the mismatch is?

**Comment by roddtalebi** [comment]
gonna checkout your branch and play with it real quick

**Comment by anishthite** [comment]
> @anishthite I had that issue but fixed it and I thought maybe your branch was created before the fix but it looks the same in the 'files changed'...hmmm. Is your [line 124](https://github.com/ezCGP/ezCGP/pull/237/files#diff-a97ccb8599becc1209701c40ab2a0ec1c167f11911abaea055690ca964dd560fR124) where you think the mismatch is?

I don't think there is a mismatch, Im just paranoid cause of your issue. As long as `get_graph_ratings()` doesn't change the refiner list (and I don't think it does) then we're set

**Comment by roddtalebi** [comment]
I'll make it an issue to verify that later then

**Comment by roddtalebi** [comment]
@anishthite I'll merge this in BUT you'll note quickly that there are some code changes that are going to affect your work:

- [added a method to establish how many objectives we have and which we are maximizing or minimizng by setting a list of boolean values...Add 'True' to the list if we want to maximize FID](https://github.com/ezCGP/ezCGP/blob/feature/simgan-support/problems/problem_simgan.py#L95)
- [got rid of the code to manually check if individual is dead to set the worst possible score](https://github.com/ezCGP/ezCGP/blob/feature/simgan-support/problems/problem_simgan.py#L110) because that check [now happens here](https://github.com/ezCGP/ezCGP/blob/feature/simgan-support/problems/problem_simgan.py#L99)

So after the merge, you'll have to make that one change to the "set_optimization_goals" method

**Pull request merged**